src/main/resources/kibana-rest.yml

```
@@ -42,19 +59,61 @@ kibana_spaces:
   versions:
     ">= 6.5.0": "/api/spaces/space"
 
+kibana_security_exception_list:
+  versions:
+    ">= 7.10.0": "/api/exception_lists/_find?namespace_type=agnostic"
+
+kibana_security_endpoint_trusted_apps:
+  versions:
+    ">= 7.10.0": "/api/endpoint/trusted_apps"
+
+kibana_security_endpoint_host_isolation:
+  versions:
+    ">= 7.14.0": "/api/exception_lists/items/_find?list_id=endpoint_host_isolation_exceptions&namespace_type=agnostic"
+
+kibana_security_endpoint_event_filters:
+  versions:
+    ">= 7.14.0": "/api/exception_lists/items/_find?list_id=endpoint_event_filters&namespace_type=agnostic"
+
+kibana_security_endpoint_list:
```

```suggestion
kibana_security_endpoint_exception_items:
```

Take Pedro's suggestion.